examples/solvers/find_optimal_parameters.py

```diff
@@ -0,0 +1,234 @@
+"""
+Example of how to use optimization in order to pick reconstruction parameters.
+
+In this example, we solve the tomographic inversion problem with different
+regularizers (fbp, huber and tv) and pick the "best" regularization parameter
+for each method w.r.t. a set of reference data.
+
+To find the "best" parameter we use Powells method to optimize a figure of
+merit, here the L2-distance to the true result.
+"""
+
+import numpy as np
+import odl
+import scipy
+
+
+def optimal_parameters(reconstruction, fom, phantoms, data,
+                       initial_param=0):
+    """Find the optimal parameters for a reconstruction method.
+
+    Notes
+    -----
+    For a forward operator :math:`A : X \to Y`, a reconstruction operator
+     parametrized by :math:`\theta` is some operator :math:`R_\theta : Y \to X`
+     such that
+
+    .. math::
+        R_\theta(A(x)) \approx x.
+
+    The optimal choice of :math:`\theta` is given by
+
+    .. math::
+        \theta = \argmin_\theta fom(R(A(x) + noise), x)
```

Comment:
These formulas are intended to be read typeset, e.g. if you click `Ctrl+I` as @kohr-h mentioned you get:

![image](https://user-images.githubusercontent.com/2202312/27162758-4aaa032c-5183-11e7-9e3f-b813d8949f56.png)

Now, this is because of something called "escaped characters", or in particular, the lack of them. Escaped characters exist because some combination of letters already carry a meaning. For example, `\t` means a tab, `\n` means a newline, etc. So if you simply write `\theta` the typesetter will read this as "[tab]eta" and it will look weird. To tell the typesetter that you truly mean "`\t`", literally, you need to write `\\t`. Here, `\\` is an escaped character indicating that you really mean the character `\\` and not any special magic to happen. This concept exists in basically all typesetting standards, including HTML, markdown, etc. Even `LaTeX` has it (try writing `%` in latex, you need to do `\%` to escape it).

If you fix this, everything typesets nicely:

![image](https://user-images.githubusercontent.com/2202312/27162906-3c81453e-5184-11e7-8477-6f0859a1c05e.png)